media/src/App.svelte

```diff
@@ -363,6 +363,7 @@
                 }
                 if (key === "shadeUp") {
                     shadeUp = val;
+                    canvas.focus();
```

Comment:
Just to be extra careful here and not crash in case the `canvas` element isn't available, I would call it like this each time:
```
if (canvas) {
    canvas.focus();
}
```

We initialize canvas as null, so it's technically possible to be calling a method on a null object somewhere throughout the lifecycle, possibly in weird edge cases, which would cause a JS error.